src/Uno.UI/DataBinding/BindingPropertyHelper.FastConvert.cs

```diff
@@ -997,6 +995,9 @@ private static bool FastStringToTypeConvert(Type outputType, string input, ref o
 				return false;
 			}
 		}
+
+		static string[] Split(string value)
+			=> Regex.Split(value, @"(\s*,\s*|\s+)");
```

Comment:
We should create the Regex only once so there's no additional parsing donc on each split:

```suggestion
      static readonly Regex _matrixSplit = new Regex(@"(\s*,\s*|\s+)");
        
		static string[] Split(string value)
			=> _matrixSplit.Split(value);
```

Reply:
I'm not using the variable specifically for matrix. Will rename it